variables.tf

```diff
@@ -17,65 +17,72 @@
 #------------------------#
 # GSuite export settings #
 #------------------------#
-variable admin_user {
+variable "admin_user" {
   description = "The GSuite Admin user to impersonate"
 }
 
-variable api {
+variable "api" {
   description = "The Admin SDK API to sync data from"
+  default     = "reports_v1"
 }
 
-variable applications {
+variable "applications" {
```

Comment:
We should consider converting this to a list of strings; the fact that it ultimately becomes a single space separated string inserted in a shell command should be abstracted away.

This can be dealt with in a separate issue.

Reply:
Openened #12